src/lib/etl.py

```diff
@@ -0,0 +1,110 @@
+"""ETL for SPOC Papers"""
+import spacy  # type: ignore
+import pandas as pd  # type: ignore
+import lxml.etree as etree  # type: ignore
+
+from typing import Any, Dict, List
+
+TEI = {"tei": "http://www.tei-c.org/ns/1.0"}
+
+
+def get_date(xml_doc: etree.Element) -> str:
```

Comment:
Does anything call this method (besides the test)?

Reply:
> Maybe a walkthrough would be in order. I don't see where some of the methods are being called. Are they called somewhere external to this repo?

@jgreben  sounds good; maybe pair after stand-up?

Reply:
My apologies, I missed where this called in your walkthrough and I do not see it in data-prep. Can you just add the comment to where it is called here. Thanks!

Reply:
Done, added comment in `process_xml` function.